site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -45,24 +45,77 @@ postman files later, you can just create a new environment again to work with
 Select the workspace menu at the top of the Postman interface, and create a new one
 called 'Egeria dojo':
 
-![Creating a new Environment 1](postman-workspace1.png)
-![Creating a new Environment 2](postman-workspace2.png)
-![Creating a new Environment 3](postman-workspace3.png)
+![Creating a new Environment 1](postman-create-workspace1.png)
+![Creating a new Environment 2](postman-create-workspace2.png)
+![Creating a new Environment 3](postman-workspace-created.png)
 
 ## Importing Egeria Postman data
 
 We now will
-* Select 'Import' and select 'GitHub' as the source
-* Authenticate as required
-* Select the "main" branch from the odpi/egeria repository on GitHub
-* Import all the Postman data found
-
-![Import Begin](postman-import-begin.png)
-![Select import source](postman-import-coderepo.png)
-![Auth with GitHub](postman-import-gotogh.png)
-![Select egeria repo](postman-import-ghrepo.png)
-![Select what to import](postman-import-select.png)
-![After Import](postman-import-after.png)
+
+* Individually import the Egeria Postman collections required for this dojo
+* Select 'Import' and select 'Link' as the source
+* Paste the URL found below for the `Egeria-admin-services-operational` collection
+
+    ```
+    https://raw.githubusercontent.com/odpi/egeria/40ec581a8dbf82eeb7e96e2412099358193e8fc1/open-metadata-implementation/admin-services/Egeria-admin-services-operational.postman_collection.json
+    ```
+
+* Confirm the correct collection will be imported
+* Confirm the collection was imported successfully
+
+![Import Begin](postman-collection-import-begin.png)
+![Select import source](postman-collection-url.png)
+![Confirm Collection to import](postman-collection-importconf.png)
+![Confirm imported successfully](postman-collection-importcomplete.png)
+
+* Repeat the same procedure for the following Egeria collections:
```

Comment:
Could we display these as a table with a column for the collection name an one for the url? If this formats poorly these we should not , but it would be worth playing with.

Reply:
A table could work nicely.

Trying to make sure the links are easily copyable would be helpful. it reduces the mouse movement etc to select.
I offered one option (which implies a quoted file) above, any ideas on more?
I think that speed of being able to copy is more important than exact format, as user has to do this a number of times

Reply:
I've been trying the table suggestion. I can set up the table however, I've tried a bunch of different approaches and from what I can find, there isn't a way to nest a "copy url" option in the table (cell content can only be on one line). I was thinking to abandon the idea of a table and instead use something like this instead: 
<img width="857" alt="Screenshot 2023-02-28 at 11 10 09 AM" src="https://user-images.githubusercontent.com/25217353/221911974-51e4c4f9-c707-4081-8486-fad9e274427b.png">

Do you think this is an acceptable approach?

Reply:
I tweaked my previous idea a bit. Instead of a collapsible section as seen above, I've given each its own section that is not collapsible since the URLs will be required. The URLs have the copy link icon as well. The collapsible sections were really only used for optional screenshots should the user want to reference.